documentation/theme/main.html

```diff
@@ -1,6 +1,25 @@
-{% extends "base.html" %}
+{% extends "base.html" %} 
 
-{% block announce %} 
-  The Pack SDK is currently in a limited beta.
-  <a href="https://coda.io/packsbeta">Click here</a> to learn more and apply.
+{% block extrahead %}
+<!-- TODO: should shape these up and should cater to specific page -->
+<meta name="description" content="Documentation for the SDK to build Coda Packs." />
+<meta name="image" content="https://cdn.coda.io/external/img/doc-banner-compressed-2x-v2.jpg" />
+
+<!-- Twitter tags -->
+<meta name="twitter:card" content="summary" />
+<meta name="twitter:site" content="@coda_hq" />
+<meta name="twitter:title" content="Coda Packs SDK" />
```

Comment:
To my chagrin, the external name was decided to be "Coda Pack SDK" (singular), so should update throughout the meta tags.

Reply:
oh that seems off to me but ok will change